config/initializers/backtrace_silencers.rb

```diff
@@ -1,6 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
 # You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
+
+# keep the lines from plugins in the backtrace
+Foreman::Plugin.modify_backtrace_cleaner
```

Comment:
Good point: I ended up in this layout mainly as evolution of the solution (I tried several ways, where the first versions needed bigger interaction with the plugin class), updated